sdk/include/xdk/potypes.h

```diff
@@ -335,6 +335,20 @@ typedef struct {
 #define CORE_PARKING_POLICY_CHANGE_ROCKET        2
 #define CORE_PARKING_POLICY_CHANGE_MAX           CORE_PARKING_POLICY_CHANGE_ROCKET
 
+#define PERFSTATE_POLICY_CHANGE_IDEAL            0
```

Comment:
Reason for moving these around?

Reply:
Because the defines were lumped between PPM constructs which that gave me an ick and I didn't like it.